src/workspaces.c

```diff
@@ -7,12 +7,12 @@
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>
+#include "buffer.h"
 #include "common/font.h"
 #include "common/graphic-helpers.h"
 #include "common/list.h"
 #include "common/mem.h"
 #include "labwc.h"
-#include "view.h"
```

Comment:
This seems unrelated to the PR

Reply:
Right, in hindsight it should have been part of #639, but I missed it then. Didn't seem worth opening a new MR for since I needed to fix up the includes in this file anyway.